package java_prgs;

import java.io.IOException;

public class exceptions {

	public static void main(String[] args) throws Exception{
		
		
			int[] arr={2,4,1};
		System.out.println(arr[3]);
		
		System.out.println("out of try/catch block");
		}
}
